### ISSUE TITLE
Does not build using `make archive` on X86 Intel

### ISSUE THREAD

**Issue by markph0204** [adobe/aepsdk-core-ios#920]
### Prerequisites

- [X] This is not a Security Disclosure, otherwise please follow the guidelines in [Security Policy](https://github.com/adobe/aepsdk-edge-ios/security/policy).
- [X] I have searched in this repository's issues to see if it has already been reported.
- [X] I have updated to the latest released version of the SDK and the issue still persists.

### Bug summary

Receive error when attempting to make XCFrameworks for AEPCore.  This fails **only** on x86_64 and works on arm64.

### Environment

Xcode 14.3
macOS: 13.3.1
architecture: x86_64 intel **fails** but arm64 **passes**

### Steps to reproduce

1. clone latest
2. make archive

### Current behavior

Output when making XCArchives: A library with the identifier 'ios-arm64' already exists.

### Expected behavior

Expect XCArchive output's to be created

### Anything else?

Not allowing me to attach build log.  Will try again after submitting issue.

[build-x86.log.zip](https://github.com/adobe/aepsdk-core-ios/files/11399331/build-x86.log.zip)

**Comment by cdhoffmann** [comment]
The log file isn't showing the error. @markph0204

**Comment by markph0204** [comment]
Improved log file attached
[build-x86.log-2.zip](https://github.com/adobe/aepsdk-core-ios/files/11401120/build-x86.log-2.zip)

**Comment by cdhoffmann** [comment]
If you run make clean and then make archive you still get this error? The only way i can replicate it is by running two make archives simultaneously without cleaning the build directory.

**Comment by markph0204** [comment]
Did a few more runs on x86_64...

1. make clean && make archive -- same issue
2. rm -rf .. git clone, etc ... -- same issue

Re checked on arm64 workstation -- works fine

**Comment by cdhoffmann** [comment]
@markph0204 Can you verify that the error is the same after running a make clean && make archive? The error log you shared can only happen if you haven't cleared the build directory.

**Comment by markph0204** [comment]
@cdhoffmann Figured it out -- both machines for Xcode 14.3 are missing tvOS.  What is interesting is the arm64 machine didn't cause error or fuss.  When I broke it down to the Xcode archive command with a few changes, the error finally displayed it was due to not having installed tvOS.

On the x86_64 I can now build this without issue.

Hmmm.. perhaps consider disabling this if not installed? We do not use tvOS as I am sure it is the least used for a lot of others too.
